docs/en/Community-Articles/2024-05-07-using-fluent-validation-with-abp/post.md

```diff
@@ -0,0 +1,95 @@
+# Using FluentValidation with ABP
```

Comment:
This can be a better title:

```suggestion
# Using FluentValidation with ABP Framework
```

Also, having an opening statement for an article is a good practise, so please add a short description for the article. It can be something like:

```
FluentValidation is a popular .NET library that allows you to define strongly typed validation rules. In this article, I will show you how you can get the benefit of this library and define custom validation rules in your application...
```